caprover/INSTALL_GC_STACK.md

```diff
@@ -70,12 +70,36 @@ You have two options to install apps:
 
 If you don't want to sweat the details, it's much quicker to deploy the Guardian Connector stack of apps using the `gc-stack-deploy`.
 
-Unless your SQL database is on another host, the tool must be run on the same machine where CapRover is running. Install the tool:
+Unless your SQL database is on another host, the tool must be run on the same machine where CapRover is running. 
 
+#### Install the tool
+
+On most systems you can install directly with pip:
+
+```sh
+$ pip install "gc-stack-deploy @ git+https://github.com/ConservationMetrics/gc-deploy.git@main#subdirectory=caprover/gc-stack-deploy"
+```
+
+#### If pip is blocked (common on some VMs)
+
+Some environments (like fresh Ubuntu VMs) restrict `pip install` into the system Python. In that case, use `pipx`
```

Comment:
You can create a venv, see the Makefile for example, and maybe that's what we want to document here.  It would not require installing `pipx`.

OTOH if Python is not installed on the VM at all (which seems somewhat common) then pipx might be a better solution. Unless it's not Debian-based and then `apt` doesn't exist.  :-(

There is no winning strategy, really.